src/makeCalculator.js

```diff
@@ -4,7 +4,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    add(num) {
+      this.result += num;
+    },
+
+    subtract(num) {
+      this.result -= num;
+    },
+
+    multiply(num) {
+      this.result *= num;
+    },
+
+    divide(num) {
+      if (num !== 0) {
+        this.result /= num;
+      }
```

Comment:
While it's good that you're checking for division by zero, consider throwing an error or returning a specific value (like `null` or `Infinity`) to handle this edge case explicitly. This way, users of the calculator will be aware that an invalid operation was attempted.